app/views/markets/show.html.erb

```diff
@@ -1,25 +1,23 @@
-<div class="container" style="height:  width:60%" >
+<div class="container">
   <h1><%= @market.name %></h1>
   <p><%= @market.category.name %></p>
   <p><%= @market.address%></p>
   <p><%= @market.description%></p>
-
-</div>
-
-<div style="width: 100%; height: 600px;">
-  data-controller="map"
-  data-map-markers-value="<%= @marker.to_json %>"
-  data-map-api-key-value="<%= ENV['MAPBOX_API_KEY'] %>"</div>
-
-
-<div>
-    <%= link_to market_reviews_path(@market) , class: "nav-link" do %>
-      Check the reviews
-    <%end%>
+  <div
+    style="width: 100%; height: 250px;"
+    data-controller="map"
+    data-map-markers-value="<%= @marker.to_json %>"
+    data-map-api-key-value="<%= ENV['MAPBOX_API_KEY'] %>"
+  >
+    <!-- this is an empty div... mapbox fills it with magic! -->
   </div>
```

Comment:
this is how the div should have been structured. notice how the `>` doesn't come after the `style` line on 7. instead, the next line sets the next attribute, and so on, until line 11.

with long tags in html, it can be helpful to split them across multiple lines.

`<img src="graphic.gif" width="100" height="60" />` is the same thing as:
```
<img
 src="graphic.gif"
 width="100"
 height="60"
/>
```